plugins/by-name/codeium-nvim/default.nix

```
@@ -15,108 +21,38 @@ lib.nixvim.plugins.mkNeovimPlugin {
     khaneliman
   ];
 
-  # TODO: added 2024-09-03 remove after 24.11
-  deprecateExtraOptions = true;
-  optionsRenamedToSettings = [
-    "configPath"
-    "binPath"
-    [
-      "api"
-      "host"
-    ]
-    [
-      "api"
-      "port"
-    ]
-    [
-      "tools"
-      "uname"
-    ]
-    [
-      "tools"
-      "uuidgen"
-    ]
-    [
-      "tools"
-      "curl"
-    ]
-    [
-      "tools"
-      "gzip"
-    ]
-    [
-      "tools"
-      "languageServer"
-    ]
-    "wrapper"
-  ];
+  description = ''
+    By default, Nixvim will install the `curl`, `gzip`, `coreutils`, `util-linux` and `codeium` packages and populate settings.tools.*` accordingly.
```

I'm struggling to come up with good wording, but I think _something_ like this is a little more explicit about what is actually happening:

```suggestion
    By default, enabling this plugin will also install the `curl`, `gzip`, `coreutils`, `util-linux` and `codeium` packages (via the `dependencies.*.enable` options`)

    Options under `settings.tools.*` with a known dependency will be configured automatically when the associated `dependencies.*.enable` option is enabled.
```

?